### DISCUSSION TITLE
Status of Unity debugger

### DISCUSSION THREAD

**Discussion by halfdan87** [mfussenegger/nvim-dap#1170] [Q&A]
Hello, guys!
What is the status of Unity debugging in neovim in 2024?
Is it doable?
What is the correct way to do it?

I have seen some discussions about it and it seems that some stuff got deprecated, the main IDEs use different technologies to communicate with Unity's debugger. I'm pretty confused. If you can share some general knowledge on the subject I will appreciate:)

**Comment by dimixar** [comment]
Same here. I'm curious to know if I can finally have a full IDE experience with Unity in nvim. The UnityDebugger configuration is what is lacking for me right now.

**Comment by nagaohiroki** [comment]
https://marketplace.visualstudio.com/items?itemName=deitry.unity-debug
Debugger for Unity 3.0.11 is working. 
(checked Unity 2022.3.39f1, Win11/macOS, nvim: v0.10.0)

vstuc(new unity debugger) is not working. 
I did a little research here.
https://github.com/mfussenegger/nvim-dap/discussions/815

**Reply by nagaohiroki** [reply]
new Unity's debugger is working.
https://github.com/mfussenegger/nvim-dap/discussions/815#discussioncomment-10745887
